modules/math/shared/src/main/scala/gsp/math/skycalc/solver/implicits.scala

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gsp.math.skycalc.solver
+
+import cats.implicits._
+import java.time.Instant
+import gsp.math.Declination
+import gsp.math.Angle
+import gsp.math.HourAngle
+import io.chrisdavenport.cats.time._
+
+object implicits {
+
+  implicit def exactGetter[A]: CalcGetter[GetterStrategy.Exact, A] =
+    new CalcGetter[GetterStrategy.Exact, A] {
+      def get[T](calc: Calculator[GetterStrategy.Exact, T])(field: T => A)(instant: Instant): A =
+        field(calc.timedResults(calc.toIndex(instant))._2)
+    }
```

Comment:
If a value isn't defined exactly at `instant`, this will return a value at the previous instant for which a value exists?  If so, should it instead be the closest instant?

Reply:
It will depend on what `toIndex` returns.

Right now, it is only used in `SingleValueCalculator` which holds a single value.

But let's do that. I was thinking of adding a `GetterStrategy.Closest` already, so let's implement it and ditch `Exact`, which may or may not be exact, depending on environment.

Also, I'm thinking of reworking things so different strategies can be specified when asking the same calculator for a value. (ie: Strategy not tied to the calculator)